otterdog/eclipse-tractusx.jsonnet

```diff
@@ -1517,5 +1517,22 @@ orgs.newOrg('eclipse-tractusx') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('wallet-dim-stub') {
+      description: "Wallet DIM stub application",
+      allow_forking: true,
+      allow_merge_commit: true,
+      dependabot_alerts_enabled: true,
+      dependabot_security_updates_enabled: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      default_branch: "main",
```

Comment:
should not be necessary, as the value defaults to main

Reply:
Addressed with: https://github.com/eclipse-tractusx/.eclipsefdn/pull/98/commits/e5bb6ba7e3fb1755b64d25973d53eb9e116cda27